src/njoy21/input/THERMR/Card2/Nbin.hpp

```diff
@@ -0,0 +1,18 @@
+struct Nbin {
+  using Value_t = int;
+  static std::string name(){ return "nbin"; }
+
+  static std::string description(){
+    return
+      "nbin is the number of equi-probable scattering angles available. For\n"
```

Comment:
The areas within each bin are not equal, but the probability of randomly picking each bin is equivalent. That's kind of saying the same thing, depending on what you integrate over.